manifests/init.pp

```diff
@@ -21,10 +21,19 @@
   # Check the client os to define the package name and service name
 
   case $::osfamily {
-    'Debian', 'Suse': {
+    'Debian': {
       $package_name = 'cron'
       $service_name = 'cron'
     }
+    'Suse': {
+      if $::lsbmajdistrelease == '12' {
```

Comment:
I would try to not use the lsb*-facts. They need extra packages to be installed which might not be available on al systems.

Better use $::operatingsystemrelease and regex to match the major version:
```if $::operatingsystemrelease =~ /^12\./```